File: xFit_rabbit/xFit_rabbit/Data/Novost.cs
using System;
using System.Collections.Generic;

namespace xFit_rabbit.Data;

public partial class Novost
{
    public int NovostID { get; set; }

    public string? Naziv { get; set; }

    public string? Sadzaj { get; set; }

    public DateTime? DatumObjave { get; set; }

    public int? KorisnikId { get; set; }

    public int? KlijentId { get; set; }

    public virtual Klijent? Klijent { get; set; }

    public virtual Korisnik? Korisnik { get; set; }
}

File: xFit/UI/xfit_mobilna/lib/models/novost.dart
import 'package:json_annotation/json_annotation.dart';

part 'novost.g.dart';

@JsonSerializable()
class Novost{
  int?novostId;
  String?naziv;
  String?sadzaj;
  DateTime?datumObjave;
  int?klijentId;

  Novost(this.novostId, this.naziv, this.sadzaj, this.datumObjave, this.klijentId);

  factory Novost.fromJson(Map<String, dynamic> json) => _$NovostFromJson(json);

  Map<String, dynamic> toJson() => _$NovostToJson(this);
}